IssueTracker/appsettings.json

```diff
@@ -13,9 +13,9 @@
     }
   },
   "ConnectionStrings": {
-    //"myconn": "server=DESKTOP-URSHK4B\\SQLEXPRESS; database=IssueTrackerDb;Trusted_Connection=True;"
+    "myconn": "server=DESKTOP-URSHK4B\\SQLEXPRESS; database=IssueTrackerDb;Trusted_Connection=True;"
     //"myconn": "server=JASONDIAS2019\\SQLEXPRESS; database=IssueTrackerDb;Trusted_Connection=True;"
-    "myconn": "Server=tcp:jdazuretest.database.windows.net,1433;Initial Catalog=AzureTest;Persist Security Info=False;User ID=jdadmin;Password=^DXpdmw94;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;"
+    //"myconn": "Server=tcp:jdazuretest.database.windows.net,1433;Initial Catalog=AzureTest;Persist Security Info=False;User ID=jdadmin;Password=^DXpdmw94;MultipleActiveResultSets=False;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;"
   },

```

Comment:
If I check this in Dev environment will break. coz it wont be able to find your local database. we have to revert this before i can checkin